### PR TITLE
Enable DataView feature by default

### PR DESCRIPTION
**Pull request by romandev**
This patch contains the following things:
  - Add a document for `DataView` feature
  - Remove NAPI_DATA_VIEW_FEATURE

Refs: https://github.com/nodejs/node-addon-api/issues/196

### CODE REVIEW

File: doc/dataview.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
+# DataView
+
```
Comment (NickNaso) [inline doc/dataview.md]: In general  as a rule in all the documentation please use the  Napi namespace e.g `Napi::DataView`.

**Comment by romandev** [comment]
@NickNaso Thank you for review. I updated the document. Please take another look.

**Comment by romandev** [comment]
@NickNaso Can you help to merge this patch?

**Comment by NickNaso** [comment]
Hi @romandev I think that we need some other approval. @mhdawson @gabrielschulhof Could you take a look at this PR? Can I provide to merge it?

**Comment by mhdawson** [comment]
Landed as b6e2d92c09d33bb4c7568b69f5bc41e6ef928d50

**Pull request closed without merging**